docs/kratos/social-signin/09_ory.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: ory
+title: Add Ory OAuth2 or Ory Hydra as a social sign-in provider
+sidebar_label: Ory
+---
+
+# Ory
+
+This **Ory** option allows you to add Ory OAuth2 as OIDC provider. In Ory Network, you can create this provider using the Ory
+Console or the Ory CLI. When self-hosting Ory, deploy the Ory Hydra Federation server and use the `hydra` CLI to create the
+client.
+
+The following snippet shows how to create it using the CLI:
+
+```bash
+ory create project --name "OAuth2 Server - Example Corp"
+# Note down the project ID
+export project_id=your-project-id # replace with your project ID
+
+ory create oauth2-client --project $project_id \
+ --name "Example Corp" \
+ --grant-type authorization_code,refresh_token \
+ --response-type code \
+ --scope openid,offline_access,email \
+ --redirect-uri https://your-project-slug.projects.oryapis.com/self-service/methods/oidc/callback/H1o_k--i # replace with your redirect URI
+```
+
+If you want to learn more head over to the [Ory OAuth2](../../oauth2-oidc/index.md) documentation.
+
+To add your Ory OAuth2 server as a social sign-in provider, you need these details:
+
+- Client ID - you get this when creating the client
+- Client Secret - you get this when creating the client
+- Issuer URL - this is the URL of the Ory Network project or Ory Hydra Federation server instance.
+
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add an Ory OAuth2 provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the **Add new OpenID Connect provider** button.
+3. Define the **Label**. This name is used for identification purposes only.
+4. Paste the configuration details obtained from your social sign-in provider into the corresponding fields in the Console:
+   - Client ID
+   - Client Secret
+   - Issuer URL
+5. Copy the Redirect URI from the Console and add it to the OAuth2 client you created earlier. You can do this in the Ory
+   Console or using the Ory CLI.
+6. Click **Save Configuration** to finish.
+
+:::note
+
+These steps cover the basic configuration of a social sign-in provider integration. At this point, the user experience is
+incomplete. To complete the configuration and ensure a smooth and secure user experience, configure the [scopes](#scopes) and
```

Comment:
Maybe phrase this positively - the basic now works. because these are separate projects, and the consuming side (3P app) may have different/additional profile fields, we have flexibility about what we copy over from the SSO. Typically you'll want to at least get their email address, but you can also request and map more fields. 
Let's configure that...

Reply:
I copy & pasted that from the other social signin pages:

- https://www.ory.sh/docs/kratos/social-signin/generic
- https://www.ory.sh/docs/kratos/social-signin/google
- https://www.ory.sh/docs/kratos/social-signin/microsoft
- https://www.ory.sh/docs/kratos/social-signin/github
- https://www.ory.sh/docs/kratos/social-signin/facebook

We can change this, but I would propose we do it in a different PR where give the whole section some love and change a lot of the older texts.

I can also leave it out, but on the other hand its important to understand why you need to add the data mapping.